src/assets/locales/ro/translation.json

```diff
@@ -1,10 +1,19 @@
 {
-  "common": {"greeting": "buna"},
+  "common": { "greeting": "buna" },
   "menu": {
     "home": "Acasa",
     "about": "Despre Practice for Good",
-    "practice-programs": "Stagii practica",
+    "practice_programs": "Stagii practica",
     "ongs": "ONG-uri",
     "contact": "Contact"
+  },
+  "about": {
```

Comment:
Aici structura poate fi imbunatatita:
"about": {
    "p4g": {
        "title": "Despre Practice for Good",
        "paragraph_1": "Lorem ...",
        "paragraph_2": "Lorem...."
    },
    "ong_hub" {
        "title": "...",
        ...
        "action": "Creaza cont"
    }
}

Asa le-am avea pe pagini->sectiuni si ar fi mult mai usor de citit